prijateli_tree/app/routers/games.py

```diff
@@ -248,57 +304,53 @@ def view_round(game_id: int, player_id: int, db: Session = Depends(get_db)):
         return {"round": current_round, "previous_answers": previous_answers}
 
 
-@router.post("/{game_id}/player/{player_id}/score")
+@router.post("/{game_id}/player/{player_id}/update_score")
```

Comment:
Since it's a post, it's implied that you are adding data. `/score/`. Technically, it should be a `put`, not a `post`.